management-app/src/constants.tsx

```diff
@@ -1 +1,22 @@
+import React from 'react';
+import AuthPage from './pages/AuthPage';
+import BoardPage from './pages/BoardPage';
+import MainPage from './pages/MainPage';
+import NotFoundPage from './pages/NotFoundPage';
+import WelcomePage from './pages/WelcomePage';
+import { TPath } from './types';
+
 export const SERVER_URL = process.env.REACT_APP_SERVER_URL || '';
+
+export const ROUTES: TPath[] = [
+  { id: 1, routePath: '/', title: 'Welcome', element: <WelcomePage /> },
+  {
+    id: 2,
+    routePath: '/main',
+    title: 'Main',
+    element: <MainPage />,
+  },
+  { id: 3, routePath: '/board', title: 'Board', element: <BoardPage /> },
+  { id: 4, routePath: '/auth', title: 'Auth', element: <AuthPage /> },
+  { id: 5, routePath: '*', title: 'Not Found Page', element: <NotFoundPage /> },
+];
```

Comment:
What do you think of this code? It gives us an option to use a variable (e.g.  'PATHES.WELCOME_PAGE' ) in navigate() method in future. Also we can use 'routePath' values as a key instead of id since they are unique.
```suggestion
const PATHES = {
WELCOME_PAGE: '/',
MAIN: '/main',
...
}

export const ROUTES: TPath[] = [
  { routePath: PATHES.WELCOME_PAGE, title: 'Welcome', element: <WelcomePage /> },
  {
    routePath: PATHES.MAIN,
    title: 'Main',
    element: <MainPage />,
  },
  { routePath: PATHES.BOARD, title: 'Board', element: <BoardPage /> },
  { routePath: PATHES.AUTH, title: 'Auth', element: <AuthPage /> },
  { routePath: PATHES.NOT_FOUND , title: 'Not Found Page', element: <NotFoundPage /> },
];
```

Reply:
I think this is a good idea and I'll implement it, but I think it is better to create some enum for ID not for path.

Reply:
done [bede5b3](https://github.com/treyer/project-management-app/pull/6/commits/bede5b3deaa250666fe4176f2b190ea8524e4769)